### PR TITLE
feat(postcards): services and services tests

### PR DESCRIPTION
**Pull request by robintsunny**
what: postcard service and tests
why: angular tutorial

### CODE REVIEW

File: .node-version [hunk 1]
```diff
@@ -1 +1 @@
-6.2.2
+6.11.1
```
Comment (ajpawlicki) [inline .node-version:1]: Why are we upgrading to this version?

Comment (robintsunny) [reply .node-version:1]: in the instructions, it says to change this to my version

File: app/services/index.js [hunk 2]
```diff
@@ -3,14 +3,16 @@
 var Angular = require('angular');
 var Cookies = require('angular-cookies');
 
-var API     = require('./api');
-var Auth    = require('./auth');
+var API = require('./api');
+var Auth = require('./auth');
+var Postcard = require('./postcard');
 var Session = require('./session');
```
Comment (ajpawlicki) [inline app/services/index.js]: nit: can you make these all consistent spacing e.g.

```node
var API      = require('./api');
var Auth     = require('./auth');
var Postcard = require('./postcard');
var Session  = require('./session');
```

Comment (robintsunny) [reply app/services/index.js]: got it

File: app/services/postcard.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+var Config = require('../../config')
+
+module.exports = function (API) {
```
Comment (ajpawlicki) [inline app/services/postcard.js:5]: nit arrow function

Comment (robintsunny) [reply app/services/postcard.js:5]: i can't use ES6 in this, it errors out

File: app/services/postcard.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+  var Postcard = {};
+
+  Postcard.url = Config.API_HOST + '/postcards';
+
+  Postcard.create = function (payload) {
```
Comment (ajpawlicki) [inline app/services/postcard.js:11]: nit arrow function

File: test/services/postcard.spec.js [hunk 5]
```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+var expect  = require('chai').expect;
+var Angular = require('angular');
+var Config  = require('../../config');
+var Sinon   = require('sinon')
+
+require('angular-mocks');
+
+describe('postcard service', function () {
```
Comment (ajpawlicki) [inline test/services/postcard.spec.js:10]: nit arrow functions in all these `describe` and `it` blocks

File: app/services/postcard.js (hunk 3 above)
Comment (ajpawlicki) [reply app/services/postcard.js:5]: lol nvm then

**Review by ajpawlicki — APPROVED** [review]
LGTM

**Review by jesscxu — APPROVED** [review]
LGTM can we try to use arrow functions and es6 string interpolation...same comments i left last time on that other PR!

File: test/services/postcard.spec.js [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+
+  var $q;
+  var $rootScope;
+  var API;
+  var Postcard;
+
+  beforeEach(Angular.mock.inject(function ($injector) {
+    $q = $injector.get('$q');
```
Comment (jesscxu) [inline test/services/postcard.spec.js]: nit: align these =

**Comment by robintsunny** [comment]
@jesscxu I tried using ES6 but then the entire project stopped functioning correctly. not sure why though, maybe node version? i can look in to it as well

**Review by vinay-pad — COMMENTED** [review]
LGTM

**Pull request merged**